connectivity/drivers/ble/FEATURE_BLE/TARGET_Ambiq_Micro/TARGET_Apollo3/AP3CordioHCITransportDriver.cpp

```diff
@@ -0,0 +1,142 @@
+/* 
+ * Copyright (c) 2020 SparkFun Electronics
+ * SPDX-License-Identifier: MIT
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+#include "AP3CordioHCITransportDriver.h"
+#include "am_mcu_apollo.h"
+#include "stdio.h"
+#include <cstring>
+
+#include "wsf_types.h"
+#include "wsf_timer.h"
+#include "bstream.h"
+#include "wsf_msg.h"
+#include "wsf_cs.h"
+
+#include "hci_drv_apollo3.h"
+
+#define PRINT_DEBUG_HCI 0
+
+#if PRINT_DEBUG_HCI
+#include "mbed.h"
```

Comment:
please include what is needed for printing, instead of mbed.h (this should be failing in CI check for mbed.h, I'll check why not).

Reply:
removed. probably wasn't caught because it is not compiled in by default.